webpack.config.js

```diff
@@ -132,24 +130,36 @@ function copy(o) {
   ];
 }
 
-function stringifyValues(obj) {
-  return Object.entries(obj)
-    .map(([k, v]) => [k, JSON.stringify(v)])
-    .reduce((o, [k, v]) => {
-      o[k] = v;
-      return o;
-    }, {});
+function transformFirefoxManifest(content) {
+  const manifest = JSON.parse(content.toString());
+
+  if (process.env.TOGGL_API_HOST) {
+    manifest.permissions = [
+      ...manifest.permissions,
+      process.env.TOGGL_API_HOST
+    ];
+  }
+
+  manifest.version = pkg.version;
+
+  return Buffer.from(JSON.stringify(manifest, undefined, 2));
 }
 
-function addVersion(content) {
-  return Buffer.from(
-    JSON.stringify(
-      {
-        version: pkg.version,
-        ...JSON.parse(content.toString())
-      },
-      undefined,
-      2
-    )
-  );
+function transformChromeManifest(content) {
```

Comment:
Rewriting this as you make DRY this code 😄 

```javascript
function transformManifest(browser) {
  return function (context) {
    // ...
    // ...
    if (browser === 'chrome') {
      manifest.externally_connectable.matches = [...]
    }
  }
}
```

Reply:
good point! I updated the PR thanks.